packages/config-utils/proxy.js

```diff
@@ -1,185 +0,0 @@
-/* eslint-disable no-console */
```

Comment:
This file canno't be deleted. Some applications (cost) are using only the proxy config from utils package. Can you keep it and only re-export the configuration? I believe that also might be the case for the rest of the deleted files. For example, the `serveLocalFile` is a separate plugin that can be added to the configuration.